README.md

```diff
@@ -56,7 +56,7 @@ Components marked `Stable` provide the following guarantees:
 - Service names and service package names will not change.
 - Service method names will not change. [from 1.0.0]
 - Service method parameter names will not change. [from 1.0.0]
-- Service method parameter types and return types will not change. [from 1.0.0] 
+- Service method parameter types and return types will not change. [from 1.0.0]
```

Comment:
Nit: unrelated change.

Reply:
My editor strips off trailing whitespace on file save. Given that this is a nit, I'm assuming it's ok for it to be merged too :)

Reply:
If so, can you mark this as resolved @tigrannajaryan? Thanks.